src/store/permissions.ts

```diff
@@ -144,4 +144,14 @@ const permissions: PermissionTree = {
 			},
 		],
 	},
+	verify: {
+		field: "memberRoleId",
```

Comment:
May be wrong on this note. Would command require everyone permissions, or are we retaining everyone being granted Member roles on joining the server? Or is permission being granted to everyone on load of src/util.ts? How is this part supposed to work?

Reply:
How this works is anyone with the verified role is no longer allowed to use the verify command.